FAQ.md

```diff
@@ -455,7 +455,7 @@ bells and whistles over the latter, but the latter executes with worst case
 linear time complexity.
 
 With that out of the way, if you've used `-P` with ripgrep, you may have
-noticed that it can be slower. The reasons for why this is are quite complex,
+noticed that it can be slower. The reasons for why this is quite complex,
```

Comment:
The previous wording was correct. Although it could probably be improved holistically.